dbConn.js

```diff
@@ -0,0 +1,9 @@
+require('dotenv').config();
```

Comment:
You can get rid of this file completely. Instead, to establish a connection, in the `server/index.js` file, add the following:

```js
const Knex = require('knex')
const { Model } = require('objection')
const KnexFile = require('../knexfile')

Model.knex(
 Knex(KnexFile[process.env.NODE_ENV])
)
```

Establishing the connection here makes it available to the whole project. Also, if the connection cannot be established, it crashes and we can gracefully handle it.

Reply:
fixed.... placed database connection in index.js....and exported it along with the server/app